plugins/postcss-dir-pseudo-class/src/index.js

```diff
@@ -29,19 +27,23 @@ function creator(opts) {
 								return;
 							}
 
-							if (':dir' !== node.value) {
+							if (':dir' !== node.value.toLowerCase()) {
+								return;
+							}
+
+							if (!node.nodes || !node.nodes.length) {
 								return;
 							}
 
 							// value of the :dir pseudo-class
-							const value = node.nodes.toString();
+							const value = node.nodes.toString().toLowerCase();
```

Comment:
This is later concatenated below which means this won't preserve developer casing. Is this something we should do?

Reply:
Good catch! I was indeed trying to avoid changing things and only using lower cased values in conditionals.

I will update this 🎉

Reply:
Coincidentally this was actually more correct imo.

- `[dir="RTL"]` or `[dir="rtl"]` is not the same.
- we generate `:not([dir="<inverse>"])`.

I think it is best to allow upper case in `:dir` and keywords but only output lowercase.